src/org/exist/xmldb/DatabaseImpl.java

```diff
@@ -303,6 +303,9 @@ private XmlRpcClient getRpcClient(final String user, final String password, fina
         final String key = user + "@" + url.toString();
         final XmlRpcClientConfigImpl config = new XmlRpcClientConfigImpl();
         config.setEnabledForExtensions(true);
+        config.setContentLengthOptional(true);
+        config.setGzipCompressing(true);
```

Comment:
not sure.... does this force GZIP compression? If so, the client need be configured for this as well.
That said... do we need client settings?  in org.exist.xmldb.DatabaseImpl#getRpcClient we need to do the same, maybe?

Reply:
(I reworked the PR)

Reply:
indeed a bit concerned on backward compatibility when reading the java doc:

https://ws.apache.org/xmlrpc/apidocs/org/apache/xmlrpc/common/XmlRpcHttpRequestConfigImpl.html#setGzipCompressing(boolean)

Reply:
@windauer `DatabaseImpl.java` is client side. You are looking at the client side settings!

Backwards compatibility is not an issue as Apache XML-RPC uses a auto-detect mechanism on the server, to detect and support whatever the client requests. It is documented here - https://ws.apache.org/xmlrpc/extensions.html

Reply:
@adamretter yes , I (not lars) checked indeed the client size; as you see, I linked to the same page earlier......